### PR TITLE
Translate the page to Portuguese

### PR DESCRIPTION
**Pull request by alexregado**
I didn't translate the words node and child because in Portuguese they cannot be translated to the original meaning. Happy to Help :)

### CODE REVIEW

File: 2-ui/1-document/03-dom-navigation/1-dom-children/solution.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+// ou
 document.body.children[1]
 ```
 
-The second `<li>` (with Pete):
+O segundo `<li>` (com o Pete):
 
 ```js
-// get <ul>, and then get its last element child
+// Obtém o <ul>, e depois seleciona o último child element
```
Comment (odsantos) [inline 2-ui/1-document/03-dom-navigation/1-dom-children/solution.md:25]:
```suggestion
// Obtém o <ul>, e depois seleciona o último elemento descendente
```

**Review by odsantos — CHANGES_REQUESTED** [review]
Hello @alexregado,

Here is my review.
Please feel free to comment where you disagree.

Would you also first pick the corresponding topic you're translating [here](https://github.com/javascript-tutorial/pt.javascript.info/issues/1),  and then change "Translate the page to Portuguese" to the title of that topic?

Thank you,
Osvaldo.

**Comment by odsantos** [comment]
Hi @alexregado , I dismissed my review.
This topic is already assigned to @exploitmik .
Would you please choose an assigned topic?
Thank you.

**Review by jjuliomarttins — APPROVED** [review]

**Comment by odsantos** [comment]
Hi @jjuliomarttins please do not approve pull requests.
Only maintainers should review and approve them. 

This article is assigned to another person, so as such it shouldn't even be approved unless the other person seemed unavailable.
Thank you.

**Comment by nazarepiedady** [comment]
@Peruibeloko, @jonnathan-ls, @odsantos, do you consider it a good idea to close this pull request?

**Comment by Peruibeloko** [comment]
@nazarepiedady lets close this one as well

**Pull request closed without merging**